src/deadline/job_attachments/README.md

```diff
@@ -33,6 +33,47 @@ RootPrefix/
 
 [ja-security]: https://docs.aws.amazon.com/deadline-cloud/latest/userguide/security-best-practices.html#job-attachment-queues
 
+### S3 Key Format
+
+#### Data Files
+
+Data files in the job attachments system are stored using a content-addressable approach. The S3 key format for data files is:
+
+```
+RootPrefix/Data/<file_hash>.<hash_algorithm>
+```
+
+For example:
+```
+my-deadline-prefix/Data/a1b2c3d4e5f6g7h8i9j0.xxh128
+```
+
+Where:
+- `<file_hash>` is the hash of the file contents
+- `<hash_algorithm>` is the algorithm used to generate the hash (currently "xxh128")
+
+This content-addressable approach ensures that identical files are only stored once, regardless of their original filenames or paths.
+
+#### Manifest Files
+
+Manifests are stored for both job inputs and task outputs. Input manifests for jobs are stored under:
```

Comment:
The statement about input manifests isn't true in general, it's just what the current job submission code does by default. Could we have this document that the location is defined by a value in the Job resource, and you can retrieve it with the Deadline CLI `deadline job get` or the AWS CLI `aws deadline get-job`? See https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_GetJob.html#API_GetJob_ResponseSyntax for the path `attachments/manifests/rootPath`.